### DISCUSSION TITLE
Possibility of using tags for status in the board view?

### DISCUSSION THREAD

**Discussion by melyso** [marcusolsson/obsidian-projects#674] [Help]
I might be missing something obvious here, but I can't seem to use tags as the status in the board view. Given how I've set up my workflow so far, this would be incredibly practical. Am I missing something, and if not, is there any plan to add this feature in the future? I get that the "tags"-field is multi-valued. However, in my case I'm using nested tags as single-valued fields. Example tags: relevance/high, writing-status/written, etc.

**Comment by Acylation** [comment]
I think you've provided a clearer use case for issue #117 . This needs some detailed designs, and I will put my thoughts there when I have the availability for this.
